client/tx/tx_test.go

```diff
@@ -421,9 +421,9 @@ func TestPreprocessHook(t *testing.T) {
 	requireT.NoError(err)
 	msg1 := banktypes.NewMsgSend(addr1, sdk.AccAddress("to"), nil)
 	msg2 := banktypes.NewMsgSend(addr2, sdk.AccAddress("to"), nil)
-	txb, err := txfDirect.BuildUnsignedTx(msg1, msg2)
+	txb, _ := txfDirect.BuildUnsignedTx(msg1, msg2)
```

Comment:
Here, for instance, I'd rather keep the error and check that it is indeed `nil`.